components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/UserDefinedAuthenticatorEndpointConfig.java

```diff
@@ -88,6 +89,8 @@ public static class UserDefinedAuthenticatorEndpointConfigBuilder {
         private String uri;
         private String authenticationType;
         private Map<String, String> authenticationProperties;
+        private List<String> allowedHeaders;
```

Comment:
No getters for the headers in UserDefinedAuthenticatorEndpointConfig level ?

Reply:
Since this is a builder class, we dont need a getter. 
Getter method in `UserDefinedAuthenticatorEndpointConfig ` class is sufficient for the requirements

Reply:
Added the getters, as it is easy to use getters directly in the runtime